pkg/util/namespace/namespace.go

```diff
@@ -23,3 +23,70 @@ func IsOpenShiftSystemNamespace(ns string) bool {
+		"openshift-cluster-machine-approver":               {},
+		"openshift-cluster-node-tuning-operator":           {},
+		"openshift-cluster-samples-operator":               {},
+		"openshift-cluster-storage-operator":               {},
+		"openshift-config":                                 {},
+		"openshift-config-managed":                         {},
+		"openshift-config-operator":                        {},
+		"openshift-console":                                {},
+		"openshift-console-operator":                       {},
+		"openshift-console-user-settings":                  {},
+		"openshift-controller-manager":                     {},
+		"openshift-controller-manager-operator":            {},
+		"openshift-dns":                                    {},
+		"openshift-dns-operator":                           {},
+		"openshift-etcd":                                   {},
+		"openshift-etcd-operator":                          {},
+		"openshift-host-network":                           {},
+		"openshift-image-registry":                         {},
+		"openshift-ingress":                                {},
+		"openshift-ingress-canary":                         {},
+		"openshift-ingress-operator":                       {},
+		"openshift-insights":                               {},
+		"openshift-kni-infra":                              {},
+		"openshift-kube-apiserver":                         {},
+		"openshift-kube-apiserver-operator":                {},
+		"openshift-kube-controller-manager":                {},
+		"openshift-kube-controller-manager-operator":       {},
+		"openshift-kube-scheduler":                         {},
+		"openshift-kube-scheduler-operator":                {},
+		"openshift-kube-storage-version-migrator":          {},
+		"openshift-kube-storage-version-migrator-operator": {},
+		"openshift-machine-api":                            {},
+		"openshift-machine-config-operator":                {},
+		"openshift-marketplace":                            {},
+		"openshift-monitoring":                             {},
+		"openshift-multus":                                 {},
+		"openshift-network-diagnostics":                    {},
+		"openshift-network-operator":                       {},
+		"openshift-oauth-apiserver":                        {},
+		"openshift-openstack-infra":                        {},
+		"openshift-operator-lifecycle-manager":             {},
+		"openshift-operators":                              {},
+		"openshift-ovirt-infra":                            {},
+		"openshift-sdn":                                    {},
+		"openshift-service-ca":                             {},
+		"openshift-service-ca-operator":                    {},
+		"openshift-user-workload-monitoring":               {},
+		"openshift-vsphere-infra":                          {},
+		"openshift-azure-operator":                         {},
+		"openshift-managed-upgrade-operator":               {},
```

Comment:
I think we need to add `openshift-azure-logging` to this list, if this is to be used in the same places as `IsOpenshiftSystemNamespace` we would lose our monitoring for our logging/mdsd pods.

Reply:
+1. I would also visually separate ARO-specific namespaces from OCP namespaces. So it is easier to update this map.